db/migrate/20171024225818_add_created_at_to_commits_keys.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatedAtToCommitsKeys < ActiveRecord::Migration
+  def change
+    add_column :commits_keys, :created_at, :datetime
```

Comment:
If you are going to be sorting by `created_at` then you should generally add an index of this column right? In our Shuttle deployment the number of CommitKeys attached to a commit goes into the thousands.